sources/model/GGRealTimeMontior.m

```diff
@@ -792,9 +804,11 @@ - (void) socketIODidConnect:(SocketIOClient *)socketIO {
     if (self.socketIOConnectedBlock) {
         
         NSLog(@"\t\thandling connect success");
-        
         self.socketIOConnectedBlock(YES, nil);
         self.socketIOConnectedBlock = nil;
+        NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                       self.developerToken, @"developer_access_token",
```

Comment:
took the above from Android : 
   public void connectCustomer(String dev_access_token, String customer_access_token, long customer_id){
        JSONObject params = new JSONObject();
        try {
            // add credentials
            if (customer_id > 0 && !TextUtils.isEmpty(customer_access_token)) {
                params.putOpt("customer_id", customer_id);
                params.putOpt("access_token", customer_access_token);
            }
            params.putOpt("dev_token", dev_access_token);
        } catch (JSONException e){
            e.printStackTrace();
        }

        // send connect broadcast
        connection.broadcast(BROADCAST_CUSTOMER_CONNECT, params);
    }
https://github.com/bringg/customer-sdk-android/blob/e96f773e5d930ec52b159e401e1111d85da37538/ordertrack/src/main/java/com/bringg/ordertrack/realtime/api/RealTimeAPIManager.java
maybe both are wrong ?